blacklist/blacklist/Common/Utils/StyleSheet.swift

```diff
@@ -10,10 +10,10 @@ import UIKit
 
 struct StyleSheet {
     // Define here fonts, colors and design constants.
-    struct Font {
-        static let robotoRegular = "Roboto-Regular"
-        static let robotoLight = "Roboto-Light"
-        static let lobsterRegular = "Lobster-regular"
+    enum Font: String {
+        case robotoRegular = "Roboto-Regular"
+        case robotoLight = "Roboto-Light"
+        case lobsterRegular = "Lobster-regular"
```

Comment:
In this case, we must declare an explicit rawValue because the identifier cannot have special characters. Do not modify this enum.